pkg/transforms/policy.go

```diff
@@ -37,6 +41,121 @@ func PolicyResourceBuilder(p *p.Policy) *PolicyResource {
 	if okns && okpp {
 		node.Properties["_parentPolicy"] = pnamespace + "/" + ppolicy
 	}
+
+	return &PolicyResource{node: node}
+}
+
+// For cert, config, operator policies.
+// This function returns `annotations`, `managedFields` for `source`,
+// and `severity`, `compliant`, `remediationAction`, `parentPolicyNs`, `parentPolicyName`
+// If `parentPolicyNs`, `parentPolicyName` do not exist, the policy is a stand-alone policy
+func getPolicyCommonProperties(c *unstructured.Unstructured, node Node) Node {
+	node.Properties["annotations"] = c.GetAnnotations()
+	node.Properties["isExternal"] = false
+
+	for _, m := range c.GetManagedFields() {
+		if m.Manager == "multicluster-operators-subscription" ||
+			strings.Contains(m.Manager, "argocd") {
+			node.Properties["isExternal"] = true
```

Comment:
In general, I'd rather surface up the data for the console to make the decision because now the logic exists in the console and search collector. I can live with this for now since it does make queries easier from search based on this value.

Reply:
@mprahl To your point, with the updated code here the goal (from @JustinKuli's review) was to reduce the fields coming from `managedFields`. Should this instead include a subset of the managed fields and let the console parse it?

Reply:
@dhaiducek I think surfacing up the manager names would likely be a good compromise as long as we don't need to filter by if a policy is external through search.

Reply:
Yeah, just manager names would be a good compromise. Maybe we can unique-ify that list, if it's not too much trouble. What I'd want to avoid is having 12 entries for the same manager just because it's changed 12 slightly different field... but I'm not really that familiar with the format, only that it seems very noisy in the openshift console when it's not auto-hidden :laughing:

Reply:
so... with only managers? like string array?

Reply:
Yeah, something like `c.GetManagedFields().map(m => m.Manager)`

Reply:
@JustinKuli's solution sounds great to me! Or even having manager, operation, subresource, and time (more or less everything but `fieldsV1`, the key that creates the noise/volume).

Reply:
I guess part of me is wondering whether, when it's pulled from Search, that the logic in the Console can still read it, so maybe leaving the structure in place might be best. I'm not sure I have the full picture of all the flows in my head though and whether that makes sense.

Reply:
(Like maybe I'm thinking too much about it and we should keep it simple like @JustinKuli's solutino 😜 )

Reply:
Can we vote? lol because Jutin's way make more simple in frontend

Reply:
@yiraeChristineKim I'm fine for now if you want to have it as "isExternal". I can let others chime in if they disagree.

Reply:
Go isExternal

Reply:
This one is interesting, this is the first case that we have pulling data from managedFields.
Would users understand the meaning of `isExternal`? Without this context, I'd assume it comes from either spec or status.

Reply:
@jlpadilla We don't use this for search-page. In Governance, we are building new tabs for configuration/operator/cert policies

Reply:
Yes, if you don't expect users to filter by this property from search, then add an underscore to hide it from the search ui.